versions/3.1.0.md

```diff
@@ -1458,12 +1455,15 @@ Binary content transferred with base64 encoding:
 
 ```yaml
 content:
-    image/png:
+    text/plain:
         schema:
             type: string
+            contentMediaType: image/png
             contentEncoding: base64
 ```
 
+Note that the `Content-Type` is `text/plain`, since from an HTTP perspective, base64-encoded data is simply text.  The `contentMediaType` and `contentEncoding` fields inform the application how to encode and decode the resource from and to its actual media type.
```

Comment:
Believe this text should be removed.

Reply:
@darrelmiller I have changed this paragraph to explain why there is a redundancy, but if you feel the new text is more confusing than helpful, I'm happy to remove it.